specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json

```diff
@@ -111,44 +111,47 @@
       "properties": {
         "diskId": {
           "description": "The disk id.",
-          "type": "string",
-          "readOnly": false
+          "type": "string"
         },
         "status": {
           "description": "The disk status.",
-          "$ref": "Disks.json#/definitions/DiskState",
-          "readOnly": false
+          "$ref": "Disks.json#/definitions/DiskState"
         },
         "sharePath": {
           "description": "The disk share path.",
-          "type": "string",
-          "readOnly": false
+          "type": "string"
         },
         "actualSize": {
           "description": "The actual size of disk.",
           "type": "integer",
           "format": "int64",
-          "readOnly": false
+          "readOnly": true
+        },
+        "provisionSize": {
+          "description": "The provision size of disk.",
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true
         },
         "managedBy": {
           "description": "Compute resource Uri which owns this disk.",
           "type": "string",
-          "readOnly": false
+          "readOnly": true
         },
-        "tenantId": {
+        "userResourceId": {
           "description": "The disk resource Uri from tenant view.",
```

Comment:
Remove "tenant" from the comment as well. Can you just search for all occurrences of "tenant" everywhere?

Reply:
Sure.